eos/b-decays/observables.cc

```diff
@@ -2108,6 +2109,27 @@ namespace eos
     }
     // }}}
 
+    // Class-I nonleptonic heavy-to-heavy
+    // {{{
+    ObservableGroup
+    make_classI_nonleptonic_heavy_to_heavy_group()
+    {
+        auto imp = new Implementation<ObservableGroup>(
+            R"(Class-I Nonleptonic Heavy-to-Heavy Decays)",
+            R"()",
+            {
+                /* B Meson Properties */
+                make_observable("B_s^0->D_s^+pi^-::BR", R"(\mathcal{B}(\bar{B}_s^0\to D_s^+\pi^-))",
+                        Unit::None(),
+                        &BqToDqPseudoscalar::branching_ratio,
+                        std::make_tuple(),
+                        { { "q", "s"} }),
```

Comment:
- ``B^0->D^+K^-::BR`` is missing as a second observable for $\bar{B}_q^0\to D_q^+ P^-$.
- Both ``B_s^0->D_s^*+pi^-::BR`` and ``B^0->D^*+K^-::BR`` are missing for $\bar{B}_q^0\to D_q^{*+} P^-$.